sneakers/src/components/DialogWithPhoto.tsx

```diff
@@ -0,0 +1,25 @@
+import { Dialog, DialogContent } from '@mui/material'
+
+export type DialogProps = {
```

Comment:
jeżeli będziesz chciał używać gdzieś jeszcze tego typu/interfejsu to lepiej jest zrobić osobny plik np w folderze types/index.ts i tma wrzucić reużywalne typy/interfejsy i jedynie importować gdzie potrzebne. Jak nie to możesz usunąć słówko export.